common/src/main/java/org/figuramc/figura/mixin/render/renderers/StuckArrowsFeatureRendererMixin.java

```diff
@@ -0,0 +1,27 @@
+package org.figuramc.figura.mixin.render.renderers;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.entity.layers.ArrowLayer;
+import net.minecraft.world.entity.Entity;
+import org.figuramc.figura.FiguraMod;
+import org.figuramc.figura.avatar.Avatar;
+import org.figuramc.figura.avatar.AvatarManager;
+import org.figuramc.figura.permissions.Permissions;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(ArrowLayer.class)
+public abstract class StuckArrowsFeatureRendererMixin {
+	private static boolean bool = true;
+
+	@Inject(method = "renderStuckItem", at = @At("HEAD"), cancellable = true)
+	private void disableStuckArrowsRendering(PoseStack matrices, MultiBufferSource vertexConsumers, int light, Entity entity, float directionX, float directionY, float directionZ, float tickDelta, CallbackInfo ci) {
+	Avatar avatar = AvatarManager.getAvatarForPlayer(FiguraMod.getLocalPlayerUUID());
```

Comment:
bad implementation, now the arrows' visibility for all players depends on the the setting of the avatar of the local player, not of their respective avatars

Reply:
yeah i fucked up and forgot to test mp, ill draft and work on fix